src/app/utils/api.js

```diff
@@ -44,3 +44,18 @@ export const post = async (path, data) => {
     throw error;
   }
 };
+
+export const edit = async (path, data) => {
```

Comment:
Let's name this function PUT, to match with the HTTP request method.

Reply:
@Chris-Maina That is a very good suggestion, but I when importing it the Saga file it creates a conflict because of the `put` effect.